tests/end_to_end/tests.py

```diff
@@ -249,6 +250,21 @@ def step_autoconf(self):
                 auto_controller.app = app
                 auto_controller._default()
 
+    def step_add_service_endpoints(self, client_id):
+        endpoint_controller = EndpointController()
+        for security_server in self.config["security_server"]:
+            configuration = endpoint_controller.create_api_config(security_server, self.config)
+            for client in security_server["clients"]:
+                for service_description in client["service_descriptions"]:
+                    for service in service_description["services"]:
+                        for endpoint in service[0]["endpoints"]:
+                            endpoint_controller.remote_add_service_endpoints(configuration, security_server, client, service_description, endpoint)
+        description = get_service_description(self.config, client_id)
+        assert len(description["services"][0]["endpoints"]) == 1
+
+        assert str(description["services"][0]["endpoints"][0]["path"]) == "/testPath"
```

Comment:
These 2 lines should be: "assert str(description["services"][0]["endpoints"][4]["path"]) == "/testPath" " and
        "assert str(description["services"][0]["endpoints"][4]["method"]) == "POST" " as there are 4 other generated endpoints which are with indexes 0 to 3 in this list. In addition the method to be asserted here should actually be "POST" not "GET", as you have placed "POST" in the test configuration yaml file.